Document solution to checkLink failing for some valid URLs (#51)
doc-detective/doc-detective.github.io: docs/get-started/actions/checkLink.md

    }
  ]
}
```

## Troubleshooting

- [`checkLink` fails due to unrecognized certificates](#checklink-fails-due-to-unrecognized-certificates)

### `checkLink` fails due to unrecognized certificates

> If the `checkLink` action fails for a URL that is valid and loads without redirects, it may be due to an internal or custom certificate that is not recognized by the testing machine.

Author: ```suggestion
If the `checkLink` action fails for a valid URL that loads without redirects, it may be due to an internal or custom certificate that the testing machine does not recognize.
```
